doc/specs/mesh.md

```diff
@@ -77,64 +84,75 @@ This guide assume you already have a Servicer properly setup and running. If not
-    "prometheus_max_open_files": 3,
-    "relay_cache_file": "data/relays.pkt",
-  	"relay_cache_background_sync_interval": -1, // time in milliseconds. https://pkg.go.dev/github.com/akrylysov/pogreb#Options
-  	"relay_cache_background_compaction_interval": 600000, // time in milliseconds. https://pkg.go.dev/github.com/akrylysov/pogreb#Options
-    "hot_reload_interval": 0, // amount of milliseconds that chains and servicers file are read again; 0 or less disable it
-    // Worker options match with: https://github.com/alitto/pond#resizing-strategies
-    // These are used:
-    // 1. interceptor of dispatch, health and height /v1/query endpoints
-    // 2. notify servicer
-    // the set of values is "repetead" for each worker per address,
-    // so if you set 20 max workers and set 10 addresses, that means 200 max_workers for notify + 20 for hooks
-    "worker_strategy": "balanced", // Kind of worker strategy, could be: balanced | eager | lazy - avoid eager if you see many timeout on servicer
-    "max_workers": 10,
-    "max_workers_capacity": 30,
-    "workers_idle_timeout": 60000,
-    "servicer_private_key_file": "key/key.json", // servicer private key to sign proof message on relay response. This should be a filename path relative to --datadir
-    "servicer_rpc_timeout": 60000, // servicer rpc timeout - greater for faraway regions like (tokyo to us east)
-    "servicer_auth_token_file": "key/auth.json", // authtoken used to call servicer. This should be a filename path relative to --datadir
-    // Servicer relay notification has a retry mechanism, refer to: https://github.com/hashicorp/go-retryablehttp
-    "servicer_retry_max_times": 10,
-    "servicer_retry_wait_min": 10, // time in milliseconds
-    "servicer_retry_wait_max": 180000 // time in milliseconds
+  "data_dir": "/home/app/.pocket/mesh",
+  "rpc_port": "8081",
+  "chains_name": "chains/chains.json",
+  "client_rpc_timeout": 30000,
+  "chains_rpc_timeout": 30000,
+  "auth_token_file": "key/auth.json",
+  "relay_cache_file": "data/relays.pkt",
+  "worker_strategy": "balanced",
+  "servicer_max_workers": 50,
+  "servicer_max_workers_capacity": 50000,
+  "servicer_workers_idle_timeout": 10000,
+  "servicer_private_key_file": "key/key.json",
+  "servicer_rpc_timeout": 60000,
+  "servicer_auth_token_file": "key/auth.json",
+  "node_check_interval": 60,
+  "session_cache_clean_up_interval": 1800,
+  "pocket_prometheus_port": "8083",
+  "prometheus_max_open_files": 3,
+  "metrics_moniker": "my-mesh-node-uid",
+  "metrics_worker_strategy": "lazy",
+  "metrics_max_workers": 50,
+  "metrics_max_workers_capacity": 50000,
+  "metrics_workers_idle_timeout": 10000,
+  "metrics_report_interval": 10,
+  "metrics_attach_servicer_label": false
 }
 ```
-3. Create Servicer private key file with following format into the path you set on `config.json`:
+3. Create Servicer private key file with one of the  following formats into the path you set on `config.json`:
+Fallback Format:
```

Comment:
Hide old format and only show as a spoiler / drop down..

Reply:
done